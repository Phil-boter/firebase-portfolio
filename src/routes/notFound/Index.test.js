import React from "react";

import { render } from "enzyme";

import NotFound from "./Index";

let language = "en";
describe("About Route", () => {
    const wrapper = render(<NotFound language={language} />);

    it("should render the NotFoundPage and match Snapshot", () => {
        expect(wrapper).toMatchSnapshot();
    });

    it("should have an h1 headerline with text 'Sorry!' to upperCase if language is en", () => {
        expect(wrapper.find("h1").text()).toEqual("SORRY!");
    });

    it("should have an h1 headerline with text 'ENTSCHULDIGE!' to upperCase if language is de", () => {
        language = "de";
        const wrapper = render(<NotFound language={language} />);
        expect(wrapper.find("h1").text()).toEqual("ENTSCHULDIGE!");
    });
});
