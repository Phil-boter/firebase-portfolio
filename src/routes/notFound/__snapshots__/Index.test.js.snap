// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Route should render the NotFoundPage and match Snapshot 1`] = `
<div
  class="notfound-container"
>
  <section
    class="notfound-section"
  >
    <h1>
      SORRY!
    </h1>
    <article>
      <p>
        The page you have requested seems not to exist. You will be redirected to Mainpage
      </p>
    </article>
  </section>
</div>
`;
